2024/door_12/index.js

```
@@ -0,0 +1,103 @@
+  });
+
+  console.log(`Rated ${rating} star(s)`);
+}
+
+
+document.addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  const ratingValue = hiddenRatingInput.value;
+  let stars = [];
+  if (ratingValue && !isNaN(ratingValue) && ratingValue > 0 && ratingValue <= 5) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)